code/game/turfs/turf.dm

```diff
@@ -535,8 +541,7 @@ GLOBAL_LIST_EMPTY(station_turfs)
 
 /turf/narsie_act(force, ignore_mobs, probability = 20)
 	. = (prob(probability) || force)
-	for(var/I in src)
-		var/atom/A = I
+	for(var/atom/A as anything in src)
```

Comment:
this is ok?

Reply:
yes?

Reply:
What if a pipe wants to react to COMSIG_ATOM_NARSIE_ACT?

Reply:
clearly, they are insulated from him (fixed)